ghec-audit-log-utils.js

```diff
@@ -71,6 +72,10 @@ function validateInput (program, config) {
     outputFile: {
       type: 'string',
       presence: false
+    },
+    source: {
+      type: 'string',
+      presence: false
```

Comment:
Use the `inclusion` validation to make sure it contains either `org` or `enterprise`